view/frontend/templates/js/replacejs.phtml

```diff
@@ -760,7 +760,9 @@ ob_start();
                         createRequest = false;
                     })
 
-            }), hints, callbacks);
+            });
+
+            var BC = await BoltCheckout.configure(promisedCart, hints, callbacks);
```

Comment:
why var BC? it seems unusued.

Reply:
I'm not sure either. I carried it over from the old code - I didn't want to do a complete overhaul of what existed.